cmd/simulator/main.go

```diff
@@ -73,12 +84,22 @@ EXIT:
 	driver.Stop()
 	clean()
 
-	log.Infof("Simulation finish. Conf: %s, TotalTick: %d, Result: %s", *confName, driver.TickCount(), simResult)
+	fmt.Printf("%s [%s] total iteration: %d, time cost: %v\n", simResult, *confName, driver.TickCount(), time.Since(start))
```

Comment:
why using fmt here?

Reply:
It's on purpose. We need to print out the simulation result no matter logs are on or off.